packages/ui/core-components/src/lib/atoms/inputs/dropdown/dropdownOptionStore.js

```diff
@@ -78,6 +78,14 @@ export const dropdownOptionStore = (multi = false, delay = 100) => {
 			if (a.idx !== b.idx) {
 				return a.idx - b.idx;
 			}
+
+				if (a.value > b.value) {
+					return 1
+				} else {
+					return -1
+				}
```

Comment:
I can't recall what this evaluates to, but it might be worth considering [`localeCompare`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare)